lib/switchboard.rb

```diff
@@ -20,5 +14,13 @@ def lookup(id)
     def base_url
       ENV['SWITCHBOARD_BASE_URL']
     end
+
+    def connection
```

Comment:
Why not use https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/lib/http_connection_factory.rb ?